src/content/docs/en/guides/migrate-to-astro/from-gitbook.mdx

```diff
@@ -0,0 +1,75 @@
+
+- Both Astro and GitBook support [Markdown](/en/guides/markdown-content/). You can migrate all your existing documentation utilizing GitBook's gitsync feature.
+
+- Both Astro and GitBook use some form of [file-based routing](/en/core-concepts/routing/). Using Astro's file structure for your existing content and when adding new pages should feel familiar.
+
+## Key Differences between GitBook and Astro
+
+When you migrate your GitBook docs to Astro, you will notice some important differences:
+
+- A GitBook site is edited using an online dashboard. In Astro, you will use a [code editor](/en/editor-setup/) and development environment to maintain your site. You can develop locally on your machine, or choose a cloud editor/development environment like StackBlitz, CodeSandbox or Gitpod.
+
+- GitBook stores your content in a database. In Astro, you will have individual files (typically Markdown or MDX) in your [project directory](/en/core-concepts/project-structure/) for each page's content. Or, you can choose to use a [CMS for your content](/en/guides/cms/), even your existing GitBook instance, and use Astro to fetch and present the data.
+
+- GitBook sites use a specialised version of Markdoc (a superset of Markdown) for content. Astro also supports Markdoc via the optional [Markdoc integration](/en/guides/integrations-guide/markdoc/) to use `.mdx` files in addition to standard Markdown files.
+
+
+## Switch from GitBook to Astro
+
+To convert a GitBook documentation site to Astro, start with our official docs theme starter template, or explore more community docs themes in our [theme showcase](https://astro.build/themes?search=&categories%5B%5D=docs).
+
+You can pass a `--template` argument to the `create astro` command to start a new Astro project with one of our official starters. Or, you can [start a new project from any existing Astro repository on GitHub](/en/install/auto/#starter-templates).
+
+  <PackageManagerTabs>
+    <Fragment slot="npm">
+    ```shell
+    npm create astro@latest -- --template docs
+    ```
+    </Fragment>
+    <Fragment slot="pnpm">
+    ```shell
+    pnpm create astro@latest --template docs
+    ```
+    </Fragment>
+    <Fragment slot="yarn">
+    ```shell
+    yarn create astro --template docs
+    ```
+    </Fragment>
+  </PackageManagerTabs>
+
+Once you have a new Astro project, you can sync your existing GitBook content to your new Astro project. GitBook has a [gitsync feature](https://docs.gitbook.com/product-tour/git-sync) that will automatically sync your GitBook content to a GitHub/GitLab repository. 
+
+To sync directly to the docs template's content collection, specify `src/content/docs/en` or `src/content/docs` as the project directory.
+
+:::caution
+When enabling git sync be sure to specify "**GitBook to GitHub**" as the priority. This will ensure that your GitBook content is synced to your GitHub repository. Otherwise, you will overwrite your existing GitBook content.
```

Comment:
```suggestion
When enabling Git Sync be sure to specify "**GitBook to GitHub**" as the priority. This will ensure that your GitBook content is synced to your GitHub repository. Otherwise, you will overwrite your existing GitBook content.
```